CONTRIBUTING.md

```diff
@@ -96,6 +96,14 @@ There is also the `./bindgen-integration` crate, which uses `bindgen` to
 generate bindings to some C++ code, and then uses the bindings, asserting that
 values are what we expect them to be both on the Rust and C++ side.
 
+The generated and expected bindings are run through `rustfmt` before they are
+compared. Make sure you have `rustfmt` up to date:
+
+```
+$ rustup update nightly
+$ rustup run nightly cargo install -f rustfmt-nightly
```

Comment:
What happens when `rustfmt` changes the default formatting? All our CI breaks until we update them? That seems not great :(

Reply:
Nope, because both the generated bindings *and* the expectations are run through `rustfmt` before being compared.